bip-0432.mediawiki

```diff
@@ -0,0 +1,141 @@
+* Multiparty smart contracts: LN BOLTs can be revamped to remove value-sapping anchors.
+* LN-Symmetry: During the contest period, no anchor value can be sourced endogenously.
+* [https://bitcoin.stackexchange.com/questions/100537/what-is-spacechain Spacechains]
+* Ark transactions, for anchors and connector outputs
+* Timeout Trees, for anchors and connector outputs
+
+===Related Work===
+
+[https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-January/021334.html SIGHASH_GROUP] style proposals are an alternative method of bringing funds to a transaction without involving CPFP by enacting a softfork. Making these pin-resistant may require follow-on policy work, or [https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-May/020458.html more general covenants] to directly stop pins we want to avoid. The drawback of these are the necessity of a softfork.
+
+[https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-September/018168.html Transaction Sponsors] is a softfork proposal to allow transactions to
+sponsor transactions with no explicit relationship in the classical UTXO
+model. Ephemeral Dust with an un-keyed anchor can be viewed as a type of opt-in transaction sponsors implemented
+purely in policy.
+
+Using a 0-value CPFP anchors is not a new idea, see:
+
+* [https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-May/015931.html LN-dev discussion on 0-value anchors]
+
+The discussion lacked a solution to the issue of the dust entering into the UTXO set
+causing negative externalities.
+
+==Definitions==
+
+Ephemeral dust: An output with dust value which is immediately spent by a child transaction.
+
+Ephemeral dust transaction: A transaction that has an ephemeral dust output.
+
+==Specification==
+
+When received by a peer for inclusion to the mempool an ephemeral dust transaction MUST:
+
+* Be an otherwise valid transaction
+* Be only considered for mining with ephemeral dust spent
+* Have only one ephemeral dust output (output values which would normally cause rejection)
+
+or will be rejected by policy. All other policy checks are left in place.
+These constraints apply only to mempool policy. Blocks are not
+invalidated by breaking these policy-only rules as they have no bearing on consensus.
+
+==Rationale==
+
+This policy is a relaxation to dust policies that most node software enforces.
+
+If a dust output never ends up unspent in a mining template, then the marginal
+exposure of the network to dust is minimized. Most identified use-cases
+of ephemeral dust only require a single dust output, so multiple dust outputs
+are left for a possible future extension.
+
+Precise rules are left to implementations. The implementation section details
+how these restrictions are enforced and under what circumstances for the known
+implementations to aid in interoperability.
+
+== Backward compatibility ==
+
+Ephemeral dust creation was previously non-standard, so there are no known conflicts
+with previous usage.
+
```

Comment:
```suggestion

== Security Considerations ==

While CPFP anchors are expected to be a main use-case for ephemeral dust, their use can pose a mining centralization risk. Since it is more space-efficient to pay a miner out-of-band to get a transaction mined than to actually use a CPFP anchor output, large miners could potentially profit from offering out-of-band fee payment services that smaller miners can't offer. In cases where this space efficiency difference is significant, such as using CPFP anchors to pay fees on small transactions, this could act to steer fee revenue to more centralized miners, a threat to Bitcoin's overall decentralization security. Out-of-band fee payment will also result in dust being added to the UTXO set.

Protocols and application developers SHOULD only use anchor outputs for fee payment as a last resort, if alternatives such as RBF with pre-signed transactions at different fee-rates are truly impractical.

```

IMO we need this section to give devs a nudge. We should put similar language in any keyless anchor BIP too.